### DISCUSSION TITLE
Potentially compatible parts.

### DISCUSSION THREAD

**Discussion by bassamanator** [bassamanator/everything-sovol-sv06#29] [General]
This discussion is a place where you can add links to parts that seem like they might serve as suitable replacements. Both tested and untested parts are fine to list here. 

If you have found a replacement part that *almost* meets specifications, but you have found a way to make it work, that's great also, please list here.

Thanks!

**Comment by albertoabrao** [comment]
Copying from another discussion, just for the record:

 The thermistor for the Ender 3 S1 works, no crimping needed. For [this](https://www.aliexpress.com/item/4000235433543.html?spm=a2g0o.order_list.order_list_main.11.6d131802HEuJWu) heatblock, it was plug-and-play.

If you have a standard CR10 heatblock, you can heat the thermistor with a heatgun, then pull the cable. I did it successfully by going up to 300C and a good, confident tug on the cable to take it out of its enclosure. [I strongly recommend using one that lets you use it as-is](https://www.aliexpress.com/item/4000235433543.html?spm=a2g0o.order_list.order_list_main.35.5cc31802fCmXzR), makes it easier - other than the longer, countersunk head screws it requires.

The heating element for the Ender 3 S1 also works, but you'll have to crimp the connector as explained on the main page of bassamanator's repo. The beauty is that the wires are just the right length after cutting the old connector out, stripping the ends, and crimping the new one. I was afraid it'd be awful, but other than making sure you have a good connection, the crimp job is made easier by the wires being stiff and short. just remember to keep the insulation close to the heating element while crimping, as it "dances" on the wire. If that doesn't make sense to you right now, it will then :)

If it helps anyone, here's the [link](https://www.aliexpress.com/item/1005005271289566.html?spm=a2g0o.order_list.order_list_main.5.6d131802HEuJWu) I used to buy my spares last time, for both thermistor and heating element.

**Comment by bassamanator** [comment]
The user @transistor1 [transistor1](https://github.com/transistor1) had created a [PR#12](https://github.com/bassamanator/everything-sovol-sv06/pull/12) that didn't quite make it into `main`. This work has some value and I didn't want it to go to waste, so I'm pasting it here. Thanks Steve!

#### Aftermarket Options

##### Heatbreaks
 | Name/Link | User | SV06-Compatible | Notes |
 | - | - | - | - |
 | [MEGA 2pcs E3D Heat Break V6 Throat M6 M7 Thread Bi-metal For E3D V6 Hotend](https://s.click.aliexpress.com/e/_DmzWJNb)  | @bassamanator | Yes | Purchased and tested. It works as well as the stock piece.<br/>*The part sold in the link could change, so make sure it has the following specs*:<img src="./images/heatbreak/aftermarket-aliexpress.png" height="200" alt='Aftermarket heatbreak'/> |
 | [2pcs Super Smooth Titanium Alloy Heatbreak](https://www.amazon.com/dp/B07JD2S4GK?ref=ppx_pop_mob_ap_share) | @transistor1 | Yes, with caveats | Purchased and tested.  It works, but it doesn't fit snugly into the extruder opening. At the time of writing, I have been using it for about 12 hours with no leaking. |
 | [Sovol 3pcs Bi-metal Heatbreak](https://www.amazon.com/dp/B0BJDPS741?psc=1&ref=ppx_yo2ov_dt_b_product_details) | @transistor1 | **NO** | Tested. **DO NOT PURCHASE**; this is incompatible with the SV06's heatbreak.  Its outer diameter is too large. |

##### Heatblocks
 | Name/Link | User | SV06-Compatible | Notes |
 | - | - | - | - |
 | [ChowThink 1-Pack High Temperature CR-10 Plated Copper Heater Block for MK8 Extruder](https://a.co/d/7dlSiH5) | @transistor1 | Yes, with caveats | Tested, but does not come with the correct screw size. Sovol SV06 extruder uses M2.5 machine screws (I believe 0.5mm pitch but not certain).  Stock screws are too short to use, so you must order some screws.  Suggestion in the Other section. |
 
##### Thermistors/Heating Cartridges
 | Name/Link | User | SV06-Compatible | Notes |
 | - | - | - | - |
 | [Sprite Extruder Pro Kit 300℃ Cartridge Heater 24V 40W + Thermistor NTC100K High Temperature Sensor](https://a.co/d/5lAXUmq) | @transistor1 | Yes, with caveats | The thermistor uses an XH1.25 JST connector, which fits the stock Sovol terminal, but the heating cartridge has an XH2.54 JST connector, which is not what the Sovol's extruder board accepts.  The extruder board uses a JST PHR-2 connector. There is a link to the connector in the Other section |

##### Other
 | Name/Link | User | SV06-Compatible | Notes |
 | - | - | - | - |
 | [Taiss 580PCS JST PH Connector kit 2.0mm 2/3/4/5/6Pin Housing,Female and Male Adapter Cable Connector](https://a.co/d/cfq421p) | @transistor1 | Yes | PHR-2 is the correct size for the SV06's heating cartridge. Requires a special crimping tool, but I was able to get the contacts on using a plier. |
 | [Hexagon Hex Socket Head Cap Allen Bolt Screw](https://a.aliexpress.com/_mOp3Dhc) | @transistor1 | Unknown | Ordered M2.5, 20mm length. Not yet tested, but hoping this will fit the ChowThink heat block. |
